virtcontainers/physical_endpoint.go

```diff
@@ -79,8 +79,16 @@ func (endpoint *PhysicalEndpoint) Attach(h hypervisor) error {
 	}
 
 	// TODO: use device manager as general device management entrance
+	var vendorID, deviceID string
+	if splits := strings.Split(endpoint.VendorDeviceID, " "); len(splits) == 2 {
```

Comment:
What if this test fails? `vendorID` and `deviceID` are added with blank values?

Reply:
Oh, I should add this empty check in `appendVFIODevice`

Reply:
I checked that I have add this empty check in govmm package
```
if isVirtioPCI[driver] {
		if vfioDev.VendorID != "" {
			deviceParams = append(deviceParams, fmt.Sprintf(",x-pci-vendor-id=%s", vfioDev.VendorID))
		}
		if vfioDev.DeviceID != "" {
			deviceParams = append(deviceParams, fmt.Sprintf(",x-pci-device-id=%s", vfioDev.DeviceID))
		}
```
so it is ok to add directly, the test also can pass(current fail is my signed-off-by not right) and I have add a new test to this.

Do you ok with this :smile:  @jodh-intel